staging/src/k8s.io/code-generator/generate-groups.sh

```diff
@@ -14,53 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o errexit
```

Comment:
This is probably the biggest breaking change for OpenShift, we currently rely on that old script in https://github.com/openshift/client-go/blob/master/hack/update-codegen.sh mostly due to limited configuration options the new script provides. I will be working on switching that over to the new script next week and figure out the gaps so we can close them together with this PR. Once I reach that point, I'll be checking how this will work with the changes from this PR.

Reply:
I also wonder who else might be affected. We deprecated those scripts in 1.28 (https://github.com/kubernetes/kubernetes/pull/117897), so maybe worth sending out a note to k-dev about that?